_posts/2019-01-23-lecture-03.md

```diff
@@ -0,0 +1,470 @@
+
+### Proposition
+
+For any Markov network $$\mathcal{G}$$ and any distribution $$P$$, we have that if $$P\models \mathcal{I}(\mathcal{G})$$ then $$P\models \mathcal{I}_l(\mathcal{G})$$.
+
+Finally, we reach at the following conclusion
+
+### Proposition
+
+Let $$P$$ be a positive distribution. If $$P$$ satisfies $$\mathcal{I}_P(\mathcal{G})$$, then $$P$$ satisfies $$\mathcal{I}(\mathcal{G})$$.
+
+### Theorem
+
+The followings are equivalent:
+
+#### $$P\models \mathcal{I}_l(\mathcal{G})$$
+#### $$P\models \mathcal{I}_P(\mathcal{G})$$
+#### $$P\models \mathcal{I}(\mathcal{G})$$
+
+## Exponential Form
+Since we don't want to constraint the clique potentials to be positive in all situations, exponential form is used to to represent a clique potential $$\phi_c(x_c)$$ in an unconstrained form using a real-value "energy" funtion $$\phi_c(x_c)$$:
+
+$$\Phi_c(x_c)=\exp\bigg\{-\phi_c(x_c)\bigg\}$$
+
+This then gives the joint probability a nice additive structure
+
+$$p(x)=\frac{1}{Z}\exp\bigg\{-\sum_{c\in C}\phi_c(x_c)\bigg\}=\frac{1}{Z}\exp\bigg\{-H(x)\bigg\}$$
+
+Where we call the sum in the exponent the "free energy":
+
+$$H(x) = \sum_{c\in C}\phi_c(x_c)$$
+
+This form of representation is is called the "Boltzmann distribution" in physics, and called a log-linear model in statstics.
+
+## Undirected Graph Exmples
+Several well-known undirected graphical models are presented in the class and as 
+follows: Boltzmann Machine (BM), Ising model, Restricted Boltzmann Machine (RBM), and Conditional Random Field (CRF).
+### Boltzmann Machine (BM)
+Boltzmann Machine is a fully connected graph with pairwise (edge) potentials on binary-valued nodes. One example is shown in the following figure:
+
+<figure class="l-body">
+    <div class="row">
+        <div class="col one">
+            <img src="{{ 'assets/img/notes/lecture-03/BM.png' | relative_url }}" />
+        </div>
+    </div>
+</figure>
+
+The probability distribution could be written as:
+
+$$p(x_1,x_2,x_3,x_4)=\frac{1}{Z}\exp\bigg\{\sum_{ij}\phi_{ij}(x_i,x_j)\bigg\}$$
+
+It could be written in a quadratic way:
+
+$$p(x_1,x_2,x_3,x_4)=\frac{1}{Z}\exp\bigg\{\sum_{ij}\theta_{ij}x_ix_j+\sum_i\alpha_ix_i+C\bigg\}$$
+
+Note that we have a $$C$$ have representing constant in the qudratic form. The over free energy function then has the form:
+
+$$H(x)=\sum_{ij}(x_i-\mu)\Theta_{ij}(x_j-\mu)=(x-\mu)^T\Theta(x-\mu)$$
+
+which can then be solved using quadratic programming.
+
+### Ising model
+In the Ising model, nodes are arranged in a regular topology (often a regular packing grid) and connected only to their geometric neighbors. It is like a sparse Boltzmann Machine. There are also multi-state Ising model, called Potts Model, in which nodes could take multiple values instead of just binary values. One example of Ising model is shown in the figure:
```

Comment:
```suggestion
In the Ising model, nodes are arranged in a regular topology (often a regular packing grid) and connected only to their geometric neighbors. It is like a sparse Boltzmann Machine. There is also the multi-state Ising model (also called Potts model), in which nodes can take multiple values instead of just binary values. One example of Ising model is shown in the following figure:
```